### PR TITLE
Feature/wmt 1007/national regional dist by grade

### PR DESCRIPTION
**Pull request by tsheppard01**
Adds national and regional case distribution by grade.  Also refectors the get-caseload routes and services

### CODE REVIEW

File: app/routes/caseload.js [hunk 1]
```diff
@@ -9,56 +10,27 @@ module.exports = function (router) {
...
+          screen: 'caseload',
+          linkId: req.params.id,
+          title: result.title,
+          subTitle: result.subTitle,
+          breadcrumbs: result.breadcrumbs,
+          subNav: getSubNav(id, organisationLevel, req.path),
+          organisationLevel: organisationLevel,
+          childOrganisationLevel: orgUnit.childOrganisationLevel,
+          childOrganisationLevelDisplayText: childOrgUnit.displayText,
+          caseloadDetails: caseloadDetails(organisationLevel, result)
```
Comment (caoimheb) [inline app/routes/caseload.js]: 👍

File: app/views/caseload.html [hunk 2]
```diff
@@ -122,22 +115,22 @@ <h3 class="heading-medium">Caseload by grade</h3>
...
-          <td headers="untiered tiers">{{ team.grades[i].untiered }}%</td>
-          <td headers="d2 tiers">{{ team.grades[i].d2 }}%</td>
-          <td headers="d1 tiers">{{ team.grades[i].d1 }}%</td>
-          <td headers="c2 tiers">{{ team.grades[i].c2 }}%</td>
-          <td headers="c1 tiers">{{ team.grades[i].c1 }}%</td>
-          <td headers="b2 tiers">{{ team.grades[i].b2 }}%</td>
-          <td headers="b1 tiers">{{ team.grades[i].b1 }}%</td>
-          <td headers="a tiers">{{ team.grades[i].a }}%</td>
+          <td headers="grade team">{{ org.grades[i].gradeCode }}</td>
+          <td headers="overall tiers">{{ ((org.grades[i].totalCases * 10)  | round) / 10 }}%</td>
```
Comment (caoimheb) [inline app/views/caseload.html]: Is there a way to write this rounding as a callable function?

Comment (tsheppard01) [reply app/views/caseload.html]: changed

File: app/views/caseload.html [hunk 3]
```diff
@@ -122,22 +115,22 @@ <h3 class="heading-medium">Caseload by grade</h3>
...
-          <td headers="d2 tiers">{{ team.grades[i].d2 }}%</td>
-          <td headers="d1 tiers">{{ team.grades[i].d1 }}%</td>
-          <td headers="c2 tiers">{{ team.grades[i].c2 }}%</td>
-          <td headers="c1 tiers">{{ team.grades[i].c1 }}%</td>
-          <td headers="b2 tiers">{{ team.grades[i].b2 }}%</td>
-          <td headers="b1 tiers">{{ team.grades[i].b1 }}%</td>
-          <td headers="a tiers">{{ team.grades[i].a }}%</td>
+          <td headers="grade team">{{ org.grades[i].gradeCode }}</td>
+          <td headers="overall tiers" class="sln-toRound">{{ org.grades[i].totalCases }}</td>
+          <td headers="untiered tiers" class="sln-toRound">{{ org.grades[i].untiered }}</td>
```
Comment (caoimheb) [inline app/views/caseload.html]: Naming convention for classes would give `sln-to-round`

File: app/views/caseload.html [hunk 4]
```diff
@@ -158,4 +151,15 @@ <h3 class="heading-medium">Caseload by grade</h3>
       $('.js-tabs').tabs();
     });
   </script>
+
+<script>
+  function roundTableValues(){
```
Comment (caoimheb) [inline app/views/caseload.html]: Move to separate javascript file. Could be used elsewhere

**Pull request merged**